addons/middle-click-popup/WorkspaceQuerier.js

```diff
@@ -1095,6 +1103,7 @@ export default class WorkspaceQuerier {
     for (const option of this.tokenGroupBlocks.parseTokens(query, 0)) {
       if (option.end >= queryStr.length) {
         if (option.isLegal) {
+          option.score += WorkspaceQuerier.SCORE_BUMP[option.type.block.id];
```

Comment:
won't this make the score of all non-bumped blocks NaN?

Reply:
good eye, good eye. I was just testing you :rofl: